.github/dependabot.yml

```diff
@@ -8,19 +8,30 @@ updates:
   - package-ecosystem: "gomod"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+    groups:
+      all-dependencies:
+        patterns:
+          - '*'
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+    groups:
+      all-dependencies:
+        patterns:
+          - '*'
   - package-ecosystem: "npm"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
     groups:
-      dependencies:
+      grafana-dependencies:
```

Comment:
Should we still group these separately?

Reply:
Given how time consuming it's been to deal even with the combined PRs, I think minimizing the number of them is probably good? But I don't have a strong opinion I don't think.